linux-bore/PKGBUILD

```diff
@@ -247,7 +247,8 @@ case "$_cpusched" in
     cachyos) # CachyOS Scheduler (EEVDF)
         source+=("${_patchsource}/sched/0001-EEVDF.patch");;
     pds|bmq) # BMQ/PDS scheduler
-        source+=("${_patchsource}/sched/0001-prjc-cachy.patch");;
+        source+=("${_patchsource}/sched/0001-prjc-cachy.patch"
```

Comment:
Maybe we should change linux-cachyos-pds.install to linux-cachyos-prjc.install (for both bmq and pds)?